public/js/src/module/records-queue.js

```diff
@@ -356,10 +398,8 @@ function _updateRecordList() {
     // rebuild the list
     return store.record.getAll( settings.enketoId )
```

Comment:
The comment for getDisplaybleRecordList implies you perhaps meant to call that function, or did I misunderstand?

Reply:
Good catch, yes!